openfl/interface/plan.py

```diff
@@ -54,6 +55,7 @@ def initialize(context, plan_config, cols_config, data_config,
     from openfl.protocols import utils
     from openfl.utilities.split import split_tensor_dict_for_holdouts
     from openfl.utilities.utils import getfqdn_env
+    from openfl.utilities.mocks import MockDataLoader
```

Comment:
Imports within functions are generally discouraged. Can we move these to the top?
I recognize this needs a fix throughout the repository.

Reply:
Fixed.